### PR TITLE
fix(su): fix bug with location name

### PR DESCRIPTION
**Pull request by WikiRik**

### CODE REVIEW

File: src/menu.json [hunk 1]
```diff
@@ -118,7 +118,6 @@
   },
   {
     "categoryName": "Summer University",
-    "auth": "logged-in",
```
Comment (serge1peshcoff) [inline src/menu.json:121]: Why removing this?

File: src/views/summeruniversity/Single.vue [hunk 2]
```diff
@@ -429,8 +435,14 @@ export default {
       this.event = response.data.data
       this.can = response.data.permissions
 
-      this.event.start_city = this.event.locations.find(location => location.start === true).name
-      this.event.end_city = this.event.locations.find(location => location.end === true).name
+      const startLocation = this.event.locations.find(location => location.start === ('true' || true))
```
Comment (serge1peshcoff) [inline src/views/summeruniversity/Single.vue:438]: Why checking if it's a string? Can there be cases like these?

File: src/menu.json (hunk 1 above)
Comment (WikiRik) [reply src/menu.json:121]: SUs are public, so it is nice that people can see it in the sidebar. The other components are still blocked by either logged-in or permissions

File: src/views/summeruniversity/Single.vue (hunk 2 above)
Comment (WikiRik) [reply src/views/summeruniversity/Single.vue:438]: Unfortunately I seems that both are in the database now. I will fix this when I fix the issue with the false value being stored

Comment (serge1peshcoff) [reply src/views/summeruniversity/Single.vue:438]: I'd rather update the DB, then this won't be needed, what do you think?

Comment (WikiRik) [reply src/views/summeruniversity/Single.vue:438]: That might be good. Can you change them all to the same type?

Comment (WikiRik) [reply src/views/summeruniversity/Single.vue]: I've changed it to the boolean true. If it's okay with you @serge1peshcoff I would like to merge this already and we can change the DB in the coming days. Is that fine?

Comment (serge1peshcoff) [reply src/views/summeruniversity/Single.vue]: I think so

**Pull request merged**